### PR TITLE
Create insertionsort.md

### PR DESCRIPTION
**Pull request by meetzaveri**
# Insertion Sort 

# [FEATURE]

## Description
Insertion Sort Algorithm which runs at O(n*n)

## What does your PR belong to?
- [x] Snippets

## Types of changes
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly. 
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by kriadmin** [comment]
Can you update it to follow the [contribution guidelines](https://github.com/kriadmin/30-seconds-of-python-code/blob/master/CONTRIBUTING.md) ?

File: snippets/insertionsort.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+### Insertion Sort
+
+```py
```
Comment (kriadmin) [inline snippets/insertionsort.md]: Change it to python because our build scrip will fail on py

File: snippets/insertionsort.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    for i in range(1, len(arr)):
+        key = arr[i]
+        j = i-1
+        while j>=0 and key < arr[j]:
+            arr[j+1] = arr[j]
+            j -= 1
+            arr[j+1] = key
+
+
+insertionsort(arr)
```
Comment (kriadmin) [inline snippets/insertionsort.md]: The example has to be on a separate block

File: snippets/insertionsort.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+print('modified %s'  %arr) # modified [2, 3, 4, 6, 7, 9]
```
Comment (kriadmin) [inline snippets/insertionsort.md]: Our snippets should not change the user's input variable and should return a values unless it is necessary to do the opposite

**Comment by kriadmin** [comment]
@meetzaveri I would really recommend you to read the contributing.md guide

**Comment by meetzaveri** [comment]
@kriadmin Sure I'll read it!

**Comment by kriadmin** [comment]
@meetzaveri I would recommend you to not change the snippet file now(except the description) and update the tag database for this snippet and also the contributor database.

**Comment by meetzaveri** [comment]
@kriadmin Okay I'll not change the snippet file. But where's tag database and contributor database?

**Comment by kriadmin** [comment]
[tag_database](https://github.com/kriadmin/30-seconds-of-python-code/blob/master/tag_database), [contributor_databse](https://github.com/kriadmin/30-seconds-of-python-code/blob/master/contributor_database)

**Comment by meetzaveri** [comment]
@kriadmin Done

**Comment by kriadmin** [comment]
Thanks for contributing! :+1:

**Pull request merged**